pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1155,8 +1155,35 @@ func (c *K8sOrchestrator) IsFSSEnabled(ctx context.Context, featureName string)
 				}
 				log.Debugf("Supervisor feature state %q in WCP cluster capabilities is set to %t", featureName,
 					supervisorFeatureState)
+
```

Comment:
Could you add a comment to explain why we are doing a deep check?

Reply:
added comment.